spec/rails_helper.rb

```diff
@@ -28,11 +28,27 @@
   config.filter_rails_from_backtrace!
 
   config.before :each do
-    DatabaseCleaner.start
```

Comment:
Was database cleaner not working as expected?/What's the benefit of the manual db clearance?

Reply:
Should be all good now 👍